backend/src/services/inventory_service.py

```
@@ -0,0 +1,1509 @@
+                "Completed inventory config validation (server-side)",
+                extra={
+                    "connector_id": connector_id,
+                    "success": success,
+                    "result_message": message,
+                    "latest_manifest": latest_manifest
+                }
+            )
+
+            return success, message
+
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)